omod/src/main/java/org/openmrs/module/pihcore/page/controller/visit/TodaysVisitsPageController.java

```diff
@@ -15,8 +18,12 @@ public void get(PageModel model,
                     @SpringBean("visitService") VisitService visitService,
                     @SpringBean("domainWrapperFactory")DomainWrapperFactory domainWrapperFactory) {
 
-        // all non-voided visits that started today
-        List<Visit> visits = visitService.getVisits(null, null, null, null, new DateMidnight().toDate(), null, null, null, null, true, false);
+        LocalDate todayDate = LocalDate.now();
+        Date startOfDayToday = java.sql.Timestamp.valueOf(LocalDateTime.of(todayDate, LocalTime.MIN));
+        Date endOfDayTomorrow = java.sql.Timestamp.valueOf(LocalDateTime.of(todayDate.plusDays(1), LocalTime.MAX));
+
+        // all non-voided visits that have been started between startOfDayToday and endOfDayTomorrow
```

Comment:
I would think this would just be endOfDayToday? In the ticket I was suggestionngateMidnight tomorrow, which is effectively the same as todayDate, LocalTime.MAX, I think?

Reply:
@mogoodrich , here are the results:
startOfDayToday: 2021-03-04 00:00:00.0
endOfDayTomorrow: 2021-03-05 23:59:59.999999999

so, which time should be, today at 23:59pm or tomorrow at 23:59pm?

Reply:
today at 23:59pm

Reply:
all set. I just updated the code. Thanks!